content/blog/2020-05-29-my-organization-s-first-r-package-workshop-at-rstudio-conf-2020/index.Rmarkdown

```diff
@@ -0,0 +1,65 @@
+---
+title: My Organization's First R Package Workshop at Rstudio::conf(2020)
+author: 
+  - "[Malcolm Barrett](https://malco.io/)"
+  - "[Richard Iannone](https://randr.rocks/)"
+date: '2020-05-29'
+categories: [learn, teach]
+tags: [conf2020]
+slug: conf20-org-first-pkg
+description: |
+  A tour of My Organization’s First R Package Workshop.
```

Comment:
"of a workshop on My Organization's First R Package" or something? the wording had me scratching my head